### ISSUE TITLE
webpack config ？

### ISSUE THREAD

**Issue by ccyytt** [CompuIves/codesandbox-client#1655]
# ❔ Question

I need to set up webapck loader and need to run the react page after setup. What should I do
### Your Environment I don't know which one to choose
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  |   
| Browser |  google
| Operating System | macOs

**Comment by ccyytt** [comment]
> # ❔ Question
> I need to set up webapck loader and need to run the react page after setup. What should I do
> 
> ### Link to sandbox: [link](https://codesandbox.io/s/1099qommkq) (optional)
> ### Environment？  I don't know which one to choose
> Software	Name/Version
> Сodesandbox	
> Browser	google
> Operating System	macOs

**Comment by lbogdan** [comment]
Hey @ccyytt ,

I don't think I understand what you want to do exactly, can you maybe rephrase / detail it a bit?

Also, we don't support webpack configuration in client sandboxes - the ones running inside the browser, so you'll probably want to use a [server sandbox](https://medium.com/@compuives/codesandbox-containers-out-of-beta-improvements-be79679df4cc).

**Comment by ccyytt** [comment]
@lbogdan  thanks

**Comment by lbogdan** [comment]
Closing this for now. If you want to discuss this further, please join us on [Spectrum](http://spectrum.chat/codesandbox).
